src/index.sass

```diff
@@ -1,13 +1,10 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+body 
```

Comment:
it's a react template we could delte these lines it's just an import statement of font globally.

Reply:
This can be done, but this is outside the scope of this PR. This PR focus on tests and workflows around tests.